scripts/clib/clib_test.py

```diff
@@ -0,0 +1,81 @@
+import os
+import argparse
+import shutil
+import subprocess
+
+def main():
+    parser = argparse.ArgumentParser(description="Test cpython library")
+    parser.add_argument("-cp", nargs=1, default="../../../cpython", required=False, help="Local cpython path.")
```

Comment:
```suggestion
    parser.add_argument("--cpython", nargs=1, default="../../../cpython", required=False, help="Local cpython path.")
```

unix commonly uses `--` for long argument names.